pkg/logger/color_handler.go

```diff
@@ -335,6 +336,10 @@ func (h *colorHandler) appendValue(buf *buffer, v slog.Value, shouldQuote bool)
 		case slog.Level:
 			h.appendLevel(buf, cv)
 		case encoding.TextMarshaler:
+			if reflect.ValueOf(cv).IsNil() {
+				h.appendString(buf, "", shouldQuote)
+				break
+			}
```

Comment:
There's also line below here that it won't let me comment on
`h.appendString(buf, fmt.Sprint(v.Any()), shouldQuote)`
instead of calling v.Any() again can you use `cv`?